libbz2-rs-sys/src/decompress.rs

```diff
@@ -1242,7 +1242,7 @@ pub(crate) fn decompress(
             }
             match current_block {
                 Block58 => {
-                    alphaSize = s.nInUse + 2;
+                    alphaSize = s.nInUse as i32 + 2;
```

Comment:
`i32::from(s.nInUse)` would clarify that this never truncates.

Reply:
in this particular case, a later commit makes `alphaSize` a `u16` too, so this is all good. `sInUse` is at most 256, and so the value `258` can never overflow the type.

But in general I agree that we probably should be using `From` more